ghc-lib-gen/src/GhclibgenFlavor.hs

```diff
@@ -42,7 +42,7 @@ ghcSeries = \case
     f | Ghc901 <= f && f < Ghc921 -> GHC_9_0
     f | Ghc921 <= f && f < Ghc941 -> GHC_9_2
     f | Ghc941 <= f && f < Ghc961 -> GHC_9_4
-    f | Ghc961 <= f && f < Ghc982 -> GHC_9_6
+    f | Ghc961 <= f && f < Ghc981 -> GHC_9_6
```

Comment:
Is this right? Why was it 9.8.2 originally?

Reply:
it's a good question. my view is that it was a mistake and clearly wrong in that it would have classified ghc-9.8.1 as a ghc 9.6 series compiler.